java/gazelle/testdata/annotations/expectedStderr.txt

```diff
@@ -0,0 +1 @@
+[90m12:00AM[0m [31mWRN[0m [1mjava/gazelle/private/maven/resolver.go:33[0m[36m >[0m not loading maven dependencies [36merror=[0m[31m"open %WORKSPACEPATH%/maven_install.json: no such file or directory"[0m [36m_c=[0mmaven-resolver
```

Comment:
It would be nice to have a record of how this was generated, so that it can be updated in the future.